### PR TITLE
remove requirements_ansible logic from the update script

### PR DESCRIPTION
**Pull request by ryanpetrello**
(`requirements_ansible` no longer exists)

### CODE REVIEW

**Review by chrismeyersfsu — APPROVED** [review]

**Review by beeankha — APPROVED** [review]
:+1:

File: requirements/updater.sh [hunk 1]
```diff
@@ -19,74 +17,36 @@ _cleanup() {
...
-  pip install "pip-tools==5.4.0"  # see https://github.com/jazzband/pip-tools/pull/1237
-}
-
-generate_requirements_v3() {
-  venv="./venv3"
-  python3 -m venv "${venv}"
+generate_requirements() {
+  venv="`pwd`/venv"
+  echo $venv
+  /usr/bin/python3 -m venv "${venv}"
```
Comment (ryanpetrello) [inline requirements/updater.sh:17]: `python` was defaulting to `awx-python` in the path, which was mixing all this up with the awx env in the image (gross).  This is more explicit and should work on EL8 systems.

File: requirements/updater.sh [hunk 2]
```diff
@@ -19,74 +17,36 @@ _cleanup() {
...
-  virtualenv -p python2 "${venv}"
-  # shellcheck disable=SC1090
-  PS1="" . "${venv}/bin/activate"
-
-  install_deps
-
-  ${pip_compile} --output-file requirements_ansible.txt "${requirements_ansible_in}"
-}
-
-generate_patch() {
```
Comment (ryanpetrello) [inline requirements/updater.sh:54]: This was cool but we thankfully don't need it anymore.

**Review by rooftopcellist — APPROVED** [review]

File: requirements/updater.sh [hunk 3]
```diff
@@ -2,15 +2,13 @@
...
-requirements_ansible_in="$(readlink -f ./requirements_ansible.in)"
 requirements="$(readlink -f ./requirements.txt)"
 requirements_git="$(readlink -f ./requirements_git.txt)"
-requirements_ansible="$(readlink -f ./requirements_ansible.txt)"
 pip_compile="pip-compile --no-header --quiet -r --allow-unsafe"
 
 check_prerequisites() {
-  for thing in patch awk python3 python2 virtualenv ; do
-      command -v $thing >/dev/null 2>&1 || { echo "$thing not installed or available. Please fix this before running." ; exit 1 ; }
+  for thing in libpq-devel; do
```
Comment (chrismeyersfsu) [inline requirements/updater.sh]: :+1: code > docs > comment

**Comment by rooftopcellist** [comment]
It looks like the license test is complaining about not finding the `click` license.
```
2021-03-11 14:58:34.265772 | static | =================================== FAILURES 
2021-03-11 14:58:34.265893 | static | _________________________ test_python_and_js_licenses 
...
2021-03-11 14:58:34.266282 | static | Exception: Included licenses not consistent with requirements:
2021-03-11 14:58:34.266327 | static |  license for requirement click is missing
```

This dependency is introduced here: https://github.com/project-receptor/receptor/blob/devel/receptorctl/setup.py#L38

File: requirements/requirements.txt [hunk 4]
```diff
@@ -1,135 +1,429 @@
...
+    # via
+    #   -r /awx_devel/requirements/requirements.in
+    #   channels
+defusedxml==0.6.0
+    # via
+    #   python3-openid
+    #   python3-saml
+    #   social-auth-core
+dictdiffer==0.8.1
+    # via openshift
```
Comment (ryanpetrello) [inline requirements/requirements.txt:85]: I wonder if we still need this dependency anymore.  I'll follow up on this.

File: requirements/requirements.txt [hunk 5]
```diff
@@ -1,135 +1,429 @@
...
+jinja2==2.11.2
+    # via
+    #   -r /awx_devel/requirements/requirements.in
+    #   openshift
+json-log-formatter==0.3.0
+    # via -r /awx_devel/requirements/requirements.in
+jsonschema==3.2.0
+    # via -r /awx_devel/requirements/requirements.in
+kubernetes==11.0.0
+    # via openshift
```
Comment (ryanpetrello) [inline requirements/requirements.txt:199]: likewise

File: requirements/requirements.txt [hunk 6]
```diff
@@ -1,135 +1,429 @@
...
+    # via
+    #   -r /awx_devel/requirements/requirements.in
+    #   django-auth-ldap
+python-string-utils==1.0.0
+    # via openshift
+python3-openid==3.1.0
+    # via social-auth-core
+python3-saml==1.9.0
+    # via -r /awx_devel/requirements/requirements.in
+pytz==2019.3
```
Comment (ryanpetrello) [inline requirements/requirements.txt:295]: Update me

File: requirements/requirements_git.txt [hunk 7]
```diff
@@ -1,3 +1,3 @@
 git+https://github.com/ansible/system-certifi.git@devel#egg=certifi
 git+git://github.com/ansible/ansible-runner@devel#egg=ansible-runner
```
Comment (ryanpetrello) [inline requirements/requirements_git.txt:2]: We should probably update this to the latest stable/major release at some point.

File: requirements/requirements_git.txt [hunk 8]
```diff
@@ -1,3 +1,3 @@
...
-git+https://github.com/project-receptor/receptor.git@#egg=receptorctl&subdirectory=receptorctl
+git+https://github.com/project-receptor/receptor.git@devel#egg=receptorctl&subdirectory=receptorctl
```
Comment (ryanpetrello) [inline requirements/requirements_git.txt]: This too.

**Comment by wenottingham** [comment]
> click

How does this work when this isn't one of our first-order dependencies?

**Comment by ryanpetrello** [comment]
> https://github.com/ansible/awx/pull/9557#issuecomment-796895085

(I discussed this privately w/ @wenottingham - it comes from receptorctl).

cc @shanemcd (when you're back around) at some point it probably makes sense for us to actually take the `receptorctl` source and release it onto PyPI as an actual named library in coordination with receptor releases, so that we can stop pulling it in here via `requirements_git`.  But that's an exercise for another day.

**Pull request merged**